app/generic/verifier/verifier.py

```diff
@@ -177,9 +176,9 @@ async def send_proof_request(
     try:
         prover = __get_verifier_by_version(proof_request.protocol_version)
 
-        await assert_valid_verifier(
-            aries_controller=aries_controller, proof_request=proof_request
-        )
+        # await assert_valid_verifier(
+        #     aries_controller=aries_controller, proof_request=proof_request
+        # )
```

Comment:
I think we only want to disable this for connectionless proof requests

```suggestion
        await assert_valid_verifier(
            aries_controller=aries_controller, proof_request=proof_request
        )
```

Reply:
yes